### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeremykubica**
Update the example code in the README.md file

### CODE REVIEW

**Review by delucchi-cmu — APPROVED** [review]
This looks good, but I'd suggest moving more example code into notebooks. Olivia is working on strategies to include notebook testing in continuous integration, so examples wouldn't fall too far behind. This is being implemented for RAIL, but would hopefully be propagated to other projects in time.

**Comment by jeremykubica** [comment]
Thanks Melissa. Most of the examples are in notebooks, which is why we missed this one. We could in theory just remove it altogether.

**Comment by jeremykubica** [comment]
It looks like there was a change to either the linter or the configs, so I needed to also update a lot of formatting throughout the project to get the presubmits to pass. It's all automatically generated changes from black.

File: README.md [hunk 1]
```diff
@@ -75,41 +75,57 @@ A short example injecting a simulated object into a stack of images, and then re
...
+# Recover the object by searching a set of trajectories.
+search = kb.stack_search(stack)
+search.search(
+    5,  # Number of search velocities to try (0, 1, 2, 3, 4)
+    5,  # Number of search angles to try (-0.1, -0.05, 0.0, 0.05, 0.1)
+    -0.1,  # The minimum search angle to test
+    0.1,  # The maximum search angle to test
+    0,  # The minimum search velocity to test
+    4,  # The maximum search velocity to test
+    7,  # The minimum number of observations
```
Comment (DinoBektesevic) [inline README.md:121]:
```python
search.search(
    vSteps=5,      # Number of search velocities to try (0, 1, 2, 3, 4)
    aSteps=4,      # Number of search angles to try (-0.1, -0.05, 0.0, 0.05, 0.1)  
    minAngle=-0.1, # The minimum search angle to test
    maxAngle=0.1,  # The maximum search angle to test
    minVelocity=0, # The minimum search velocity to test
    maxVelocity=4, # The maximum search velocity to test
    minObservations=7 # The minimum number of observations
)
```
I will just note that the above also doesn't seem right, the signature of the function looks to be
https://github.com/dirac-institute/kbmod/blob/25456c6e048815fc9f8a56fa60802c33a84233ff/src/kbmod/search/KBMOSearch.h#L44
so the angles and velocity comments above seem flipped.

**Review by DinoBektesevic — APPROVED** [review]
Super confusing with the appearance of black differences, but otherwise approved.

File: README.md (hunk 1 above)
Comment (jeremykubica) [reply README.md]: You are correct. I changed the comments to match.

**Pull request merged**